dev-lang/nasm/nasm-2.14.03rc2.recipe

```diff
@@ -34,6 +34,8 @@ BUILD_PREREQUIRES="
 	cmd:ld
 	cmd:make
 	cmd:sed
+	cmd:autoreconf
```

Comment:
Why add autoreconf if you're not using it?
```diff
- 	cmd:autoreconf
```

Reply:
At first, I thought the compile errors was because of config file, as the compiler flags in 14.02 are different, and modifying `config.ac` requires running `autoreconf`.
Now it is not needed, and I will remove it.